### DISCUSSION TITLE
Tuning Parameters used for CC and GCH

### DISCUSSION THREAD

**Discussion by Nellytado** [NatLabRockies/floris#899] [Q&A]
Hi, 

I am trying to understand the concept's behind the CC and GCH model. In the brief tutorial (chrome-extension://efaidnbmnnnibpcajpcglclefindmkaj/https://www.nrel.gov/wind/assets/pdfs/systems-engineering-workshop-2019-floris.pdf)  it says the GCH 4 tuning parameters are used and in the CC model only 2 tuning parameters are used.  However, in the yaml files for each in FLORIS v3 I can see the same amount of parameters used for the flow field:

  reference_wind_heighT,
  turbulence_intensity,
  wind_directions,
  wind_shear,
  wind_speeds,
  wind_veer,

So what does it mean that only two tuning parameters are used in CC and why would this be an advantage compared to GCH? 

Any help is most appreciated as I am trying to understand FLORIS to use for my final year project!

**Comment by bayc** [comment]
Hello @Nellytado 

I am assuming that by "CC" you mean the curled wake model that is presented in that tutorial from 2019? If so, then that model implementation had two main parameters to tune the wake deficits to best match training data. However, the curled wake model hasn't been implemented in FLORIS since version v2.4. I would highly recommend you use the latest version of FLORIS at this time, v4.0.1.

By tuning parameters, we are referring to the input paramters to the wake models themselves (e.g. for the Gauss model, line [255](https://github.com/NREL/floris/blob/main/examples/inputs/gch.yaml#L225) of our example input file). Those are the 4 tuning parameters for the Gaussian wake model. The parameters you list above all have to do with the wind condition itself, and not the wake model tuning parameters.

**Reply by Nellytado** [reply]
Ok thank you. Is there anywhere I can get more information on tuning parameters as I am confused what they are. Also I tried to access some information on how the grid solver works in FLORIS but there is an error when I access the page 

Website 
https://nrel.github.io/floris/floris_101.html. 

Image 
![image](https://github.com/NREL/floris/assets/165584201/2b384e50-0836-4abf-ab2d-c71acb3b3de3)

**Reply by rafmudaf** [reply]
@Nellytado it has moved to https://nrel.github.io/floris/intro_concepts.html. Note the main page for the FLORIS documentation is https://nrel.github.io/floris/.
